### PR TITLE
Fire a location-changed event whenever we update the URL.

### PR DESCRIPTION
**Pull request by rictic**
Supports multiple iron-page-url elements working without direct coordination.

(I thought this was there already? I guess anything that isn't tested doesn't exist :)

### CODE REVIEW

**Comment by rictic** [comment]
I also switched to using test-fixture rather than calling lifecycle methods manually like an animal.

File: test/iron-page-url.html [hunk 1]
```diff
@@ -64,11 +78,7 @@
     suite('when used solo', function() {
       var urlElem;
       setup(function() {
-        urlElem = document.createElement('iron-page-url');
-        urlElem.attached();
-      });
-      teardown(function() {
-        urlElem.detached();
+        urlElem = fixture('Solo');
```
Comment (cdata) [inline test/iron-page-url.html]: :beers:

**Comment by cdata** [comment]
LGTM :+1:

**Comment by rictic** [comment]
Woot, thanks for the review!

**Pull request merged**